src/openzaak/conf/base.py

```diff
@@ -332,17 +417,25 @@
     "content-crs",
 )
 
-# Raven
-SENTRY_DSN = os.getenv("SENTRY_DSN")
+#
+# DJANGO-PRIVATES -- safely serve files after authorization
+#
+PRIVATE_MEDIA_ROOT = os.path.join(BASE_DIR, "private-media")
+PRIVATE_MEDIA_URL = "/private-media/"
+
+# requires an nginx container running in front
+SENDFILE_BACKEND = getenv("SENDFILE_BACKEND", "sendfile.backends.nginx")
+SENDFILE_ROOT = PRIVATE_MEDIA_ROOT
+SENDFILE_URL = PRIVATE_MEDIA_URL
+
+#
+# RAVEN/SENTRY - error monitoring
+#
+SENTRY_DSN = getenv("SENTRY_DSN")
 
 if SENTRY_DSN:
     INSTALLED_APPS = INSTALLED_APPS + ["raven.contrib.django.raven_compat"]
 
-    if "GIT_SHA" in os.environ:
```

Comment:
This should return. Steven moved it here because it could crash if no SENTRY/DSN was present.

Reply:
Fixed in f91534797675e1bd4d5febc321fa048a57866edd